.vscode/settings.json

```diff
@@ -0,0 +1,29 @@
+{
+    // Always remove trailing whitespaces
+    "files.trimTrailingWhitespace": true,
+    "[python]": {
+        "editor.tabSize": 4,
```

Comment:
Shall we add `"editor.formatOnSave": true` or not?

Reply:
Great point. Eventually I think we should. The current challenge is that if a file is auto formatted, the formatted change may dilute the actual change in a PR. What do you think?

Reply:
In some degree, it could enforce that python script been formatted. It has limitation like it is not working when AutoSave on delay is enabled.

Reply:
Sounds good. I added the line.